font-test-data/src/ift.rs

```diff
@@ -4,6 +4,7 @@
 //! <https://w3c.github.io/IFT/Overview.html>
 
 use std::collections::HashMap;
+use std::iter::repeat;
```

Comment:
very much a nit, but a rust idiom I've gotten kind of into is not to import free fns directly but to import the containing module, and then call the fn through that, e.g. do `iter::repeat(0)` instead of just `repeat(0)`; I find it makes for easier reading?

anyway not a blocker obviously, just a note..